openassessment/xblock/static/sass/xb/elements/_controls.scss

```diff
@@ -146,74 +111,45 @@
   }
 
   &.disabled, &.is--disabled, &[disabled] {
-    opacity: 0.4;
+    color: $action-primary-color-disabled;
+    border-color: $action-primary-color-disabled;
+    background: $action-primary-color-disabled-back;
   }
 }
 
 
-// buttons - secondary (assuming dark bg/light copy)
+// buttons - secondary
 %btn--secondary {
   @extend %btn-pill;
   @extend %t-strong;
   padding: ($baseline-v/2) ($baseline-h);
-  background: $action-secondary-color;
-  border: 1px shade($action-secondary-color, 10%);
-  color: tint($action-secondary-color, 90%);
+  background: $white;
```

Comment:
Secondary button is white blackground with blue border/text, reverses on hover. Disabled is a grayish also (from pattern library).